sidebars.js

```diff
@@ -172,7 +172,6 @@ const sidebars = {
       },
       items: [
         "motoko/version",
```

Comment:
```suggestion
        "motoko/version",
        "motoko/main/motoko",
```
I wonder if it makes sense to also have a duplicate link here, just as an intro to the following docs. I'll leave it up to you. You can just commit the suggestion if you want to change, or leave it if not.

Reply:
I agree in principal, but doing so causes two items to be highlighted at the same time:
<img width="1379" alt="Screenshot 2023-03-07 at 12 22 04 PM" src="https://user-images.githubusercontent.com/98767015/223543498-9e3509fd-5b8b-472f-b4cb-c928b17f49e1.png">